src/jti_background.js

```diff
@@ -44,7 +44,8 @@ if (browserType !== 'Firefox') {
 // -------------------------------------------------------------------------- //
 
 var StorageID = 'Jira-Template-Injector';
-var emptyData = {'options': {'limit': []}, 'templates': {}};
+var rateStatus = 'JTI-Rate-Status';
```

Comment:
You might need to update this to something like:
`var storageTogglesKey = 'toggles';